laravel/app/Models/User.php

```diff
@@ -1,25 +0,0 @@
-<?php
-
-namespace App\Models;
```

Comment:
I think both laravel/config/auth.php and laravel/config/services.php reference the User model. Not sure if they need to be deleted too.

Reply:
nice catch Spencer!

Reply:
Since we don't use the features of laravel that the config options are used for (authentication and stripe (which I think is an API for accepting digital payment?)), the references are unused (and therefore, irrelevant to the function of the application).

We can safely remove them if y'all would prefer, but they aren't doing any harm.